.changeset/rude-peaches-lick.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
actually - right now this one should probably be removed because it will be counted towards a release but we do not release `@xstate/react` with changesets (yet)